iatikit/standard/codelist_mappings.py

```diff
@@ -89,6 +89,8 @@ def get_path(value):
         error_log = []
         for mapping in mappings:
             xpath_query, codelist = parse_mapping(mapping)
+            if not codelist.complete:
```

Comment:
@andylolz   I also did a pull request which is same as yours but did not push it due to this.

Doing 
```
if codelist and not codelist.complete:
```
here means the test pass. The `None` codelist is the `Version` which we do not care about as we are doing validation on the `iati-activity` level .  However we should add a comment or somthing to make that explicit as to why a `codelist` could be none.

Reply:
The only reason this did not get picked up before is that no `iati-activity` have version in it so the fact the codelist is `None` never got picked up.

Reply:
> Doing
> 
> ```
> if codelist and not codelist.complete:
> ```
> here means the test pass

Oh, that makes sense – yeah, that’s a good hack for getting the tests passing.

> The `None` codelist is the `Version` one

Sorry, I don’t follow I’m afraid. What do you mean by “The `None` codelist”? Can you provide a link to some code that might help clarify?

Reply:
Aha! Sorry right, yes, I see what you mean. You’re absolutely right! But I think we can fix it by changing the fixture.

Reply:
Not sure we can change the fixture as in the real mapping file the `Version` mapping does exist causing the same problem.

Reply:
> Not sure we can change the fixture as in the real mapping file the `Version` mapping does exist causing the same problem.

I don’t think so… Can you provide steps to reproduce?

The mapping I’ve removed is an `iati-organisation` mapping, so should never have been present in the activity-mappings.json file.

If you run:

```python
iatikit.download.standard()
```

…and check in `__iatikitcache__/standard/codelists`, you should find a Version.json file. So this problem shouldn’t occur IRL.

The problem only occurs during testing because the codelist fixtures are a subset of all codelists.

Reply:
> Not sure we can change the fixture as in the real mapping file the `Version` mapping does exist causing the same problem.

I don’t think so… The mapping I’ve removed is an `iati-organisation` mapping, so should never have been present in the activity-mappings.json file.

If you run:

```python
iatikit.download.standard()
```

…and check in `__iatikitcache__/standard/codelists`, you should find a Version.json file. So this problem shouldn’t occur IRL.

The problem only occurs during testing because the codelist fixtures are a subset of all codelists.

If you think I’ve got this wrong (entirely possible!) please could you provide steps to reproduce?